docker-compose-edinburgh-1.0.0.yml

```diff
@@ -77,7 +77,7 @@ services:
       - consul
 
   vault:
-    image: edgexfoundry/docker-edgex-vault:1.0.0
+    image: edgexfoundry/docker-edgex-vault:1.0.0-dev
```

Comment:
Probably don't want the `-dev` tag in here. That's for local docker-compose work.

Reply:
will remove the -dev tag.